outsite.nl/components/pages/barbuddy/BarbuddyForm.vue

```diff
@@ -12,19 +12,23 @@ const props = defineProps({
   barBuddies: { type: Array, required: true },
 })
 
-const form = useReMemberForm('barbuddy', {
+const form = useReMemberForm('barbuddy2', {
   name: '',
   age: '',
   language: locale.value === 'nl' ? 'dutch' : 'english',
   email: '',
   phone_number: '',
   pronouns: '',
-  barbuddy: 'no_preference',
+  buddy_preferences: 'no_preference',
+  buddies: [],
   remarks: '',
 })
 
 const formElement = ref(null)
 const submit = async () => {
+  if (form.fields.buddies.length > 0) {
+    form.fields.buddy_preferences = form.fields.buddies.join(', ')
+  }
   await form.submit()
```

Comment:
```suggestion
  await form.submit({
    buddy_preferences: form.fields.buddies.join(', ') || 'no_preference'
  })
```

I'd do something like this and remove the previous 3 lines + the property on line 22